site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -325,6 +325,24 @@ The password minimum age in seconds. When enabled FusionAuth will not allow a pa
 [field]#tenant.minimumPasswordAge.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.8.0#::
 Indicates that the minimum password age is enabled and being enforced.
 
+[field]#tenant.multiFactorConfiguration.authenticator.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `true`# [since]#Available since 1.26.0#::
```

Comment:
let's change all the user displayed `two-factor` text to `multi-factor` to keep it in line with the section title.